website/docs/r/edge_cluster.html.markdown

```diff
@@ -31,6 +31,7 @@ The following arguments are supported:
 * `description` - (Optional) Description of the resource.
 * `tag` - (Optional) A list of scope + tag pairs to associate with this resource.
 * `edge_ha_profile_id` - (Optional) Edge high availability cluster profile ID.
+* `failure_domain_allocation` - (Optional) Flag to enable failure domain based allocation. Enable placement algorithm to consider failure domain of edge transport nodes and place active and standby contexts in different failure domains.
```

Comment:
We need to provide supported values here, otherwise the user might think its hcl bool

Reply:
Done.